.github/workflows/all.yml

```diff
@@ -5,14 +5,6 @@ permissions:
 
 on:
   workflow_dispatch:
-  merge_group:
-    paths:
-      - '.github/workflows/all.yml'
-      - '**.gradle'
-      - 'tools/gradle/**'
-      - 'gradlew'
-      - 'gradlew.bat'
-      - '.github/actions/**'
```

Comment:
Removing merge group action from the all workflow as it takes an awful lot of time to complete [19 min from a recent run](https://github.com/build-trust/ockam/actions/runs/4304773329).